README.rst

```diff
@@ -59,29 +59,29 @@ This plugin is using the following error codes:
 +--------------------------------------------------------------------+
 | Presence of implicit parameters                                    |
 +------+-------------------------------------------------------------+
-| P101 | format string does contain unindexed parameters             |
+| STRF101 | format string does contain unindexed parameters             |
 +------+-------------------------------------------------------------+
-| P102 | docstring does contain unindexed parameters                 |
+| STRF102 | docstring does contain unindexed parameters                 |
 +------+-------------------------------------------------------------+
-| P103 | other string does contain unindexed parameters              |
+| STRF103 | other string does contain unindexed parameters              |
 +------+-------------------------------------------------------------+
 | Missing values in the parameters                                   |
 +------+-------------------------------------------------------------+
-| P201 | format call uses too large index (INDEX)                    |
+| STRF201 | format call uses too large index (INDEX)                    |
 +------+-------------------------------------------------------------+
-| P202 | format call uses missing keyword (KEYWORD)                  |
+| STRF202 | format call uses missing keyword (KEYWORD)                  |
 +------+-------------------------------------------------------------+
-| P203 | format call uses keyword arguments but no named entries     |
+| STRF203 | format call uses keyword arguments but no named entries     |
 +------+-------------------------------------------------------------+
-| P204 | format call uses variable arguments but no numbered entries |
+| STRF204 | format call uses variable arguments but no numbered entries |
 +------+-------------------------------------------------------------+
-| P205 | format call uses implicit and explicit indexes together     |
+| STRF205 | format call uses implicit and explicit indexes together     |
 +------+-------------------------------------------------------------+
 | Unused values in the parameters                                    |
 +------+-------------------------------------------------------------+
-| P301 | format call provides unused index (INDEX)                   |
+| STRF301 | format call provides unused index (INDEX)                   |
 +------+-------------------------------------------------------------+
-| P302 | format call provides unused keyword (KEYWORD)               |
+| STRF302 | format call provides unused keyword (KEYWORD)               |
 +------+-------------------------------------------------------------+
```

Comment:
Every line, also the horizontal seperator, needs to have the same width as otherwise the table will be broken. For example Github doesn't even render any of this.